doc/source/skips/template.rst

```diff
@@ -0,0 +1,83 @@
+.. _skip-template::
+
+==============================
+SKIP Template and Instructions
+==============================
+
+:Author: <list of authors' real names and, optionally, email addresses>
+:Status: <Draft | Active | Accepted | Deferred | Rejected | Withdrawn |
+          Final | Superseded>
+:Type: <Standards Track | Process>
+:Created: <date created on, in yyyy-mm-dd format>
+:Resolved: <date resolved, in yyyy-mm-dd format>
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+:Version effective: <version-number> (for accepted SKIPs)
+
+Abstract
+--------
+
+The abstract should be a short description of what the SKIP will achieve.
+
```

Comment:
Note that NEP 0 has been updated with a "Motivation & Scope" section that I think is a very helpful addition.  Perhaps just grab a new copy of the NumPy template for now.

Reply:
What do you mean by "just grab a new copy"? This document contains a *lot* of changes from the NumPy one.

Reply:
I guess then just grab that paragraph.  Motivation and scope is important.

Reply:
:+1: done